Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -788,6 +786,12 @@ void WebGLRenderingContextBase::initializeNewContext()
     m_maxCubeMapTextureSize = m_context->getInteger(GraphicsContextGL::MAX_CUBE_MAP_TEXTURE_SIZE);
     m_maxCubeMapTextureLevel = WebGLTexture::computeLevelCount(m_maxCubeMapTextureSize, m_maxCubeMapTextureSize);
     m_maxRenderbufferSize = m_context->getInteger(GraphicsContextGL::MAX_RENDERBUFFER_SIZE);
+    m_context->getIntegerv(GraphicsContextGL::MAX_VIEWPORT_DIMS, m_maxViewportDims);
+
+    if (isWebGL2())
+        m_isDepthStencilSupported = m_context->isExtensionEnabled("GL_OES_packed_depth_stencil"_s) || m_context->isExtensionEnabled("GL_ANGLE_depth_texture"_s);
+    else
+        m_isDepthStencilSupported = m_context->isExtensionEnabled("GL_EXT_packed_depth_stencil"_s) || m_context->isExtensionEnabled("GL_ANGLE_depth_texture"_s);
```

Comment:
Could we do
```
    auto packed_depth_stencil_ext = isWebGL2() ? "GL_OES_pack..."_s : "GL_EXT_pack..."_s;
    m_isDepthStencilSupported = m_context->isExtensionEnabled(packed_depth_stencil_ext) || m_context_isExtensionEnabled("GL_ANGLE_dept..."_s);
```
instead?

Reply:
Even simpler: both context versions should check for the `_OES_` string. The `_EXT_` does not exist in ES anyway.

Reply:
done